app/serializers/school_base_serializer.rb

```diff
@@ -1,5 +1,5 @@
 class SchoolBaseSerializer < BaseSerializer
 
-  attributes :id, :address_city, :address_one, :address_state, :address_two, :address_zip, :name, :website
```

Comment:
@unzunz can you see why it's better to move these attributes into the `SchoolShowSerializer`? Hint: it has to do with what we render on the client-side.